lib/main/atom/views/symbols/fileSymbolsView.tsx

```diff
@@ -16,9 +17,12 @@ export async function toggle(editor: TextEditor, deps: Deps) {
     }
     const tag = await selectListView({
       items: generateFile(filePath, deps),
-      itemTemplate: ({name, position}) => (
+      itemTemplate: ({name, position}, ctx) => (
         <li class="two-lines">
-          <div class="primary-line">{name}</div>
+          <HighlightComponent
+            styleClass="primary-line"
+            matches={utils.highlightMatches(name, ctx.getFilterQuery())}
```

Comment:
I'm pretty sure we're not going to use `HighlightComponent` with any matching engine besides `highlightMatches`. Maybe better have `name` and `query` attributes on `HighlightComponent` and push `hightlightMatches` call inside?
Also while at it, I would prefer to have `HighlightComponent` to render as an inline element to have more flexibility, since it might be a good idea to reuse it in 'go to declaration` etc. So
```tsx
<div class="primary-line">
<HighlightComponent ... />
</div>
```
is wordier, but is more flexible then what you have here.